packages/ia-components/sandbox/details/DetailsDownloadOptions.jsx

```diff
@@ -71,26 +77,31 @@ export default class DetailsDownloadOptions extends React.Component {
             >
               {formats('format', k).downloadable}
               {' '}
-              <I18nIcon className="iconochive-download" en="download"/>
+              <I18nIcon className="iconochive-download" en="download" />
             </AnchorDownload>
+            {(typeof DwebArchive === 'undefined' || !DwebArchive.mirror || (k !== 'Epub')) ? null : (
+              <a href={`/epubreader/index.html?bookPath=/download/${this.props.identifier}/${downloadableFilesDict[k][0].metadata.name}`} title="Read online">
+                <I18nIcon className="iconochive-eye" en="Read online" />
+              </a>
+            )}
           </div>
         ))}
         <div className="show-all">
-          {(this.props.disconnected) ? null :
+          {(this.props.disconnected) ? null : (
```

Comment:
this can be more succinct with `!this.props.disconnected && <dom>` - for better readability. or can be abstracted to function